### PR TITLE
[PorteinSolvation] Updated doc code snippets

### PR DESCRIPTION
**Pull request by david451lima**
Updates the code snippets present in the documentation - now it works when copied.
Also disponibilized a PDB file for the user to imediately test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,18 +43,18 @@ And you are good to go! You can start [exploring our tutorial notebooks](https:/
...
-Example of how to use a scenario:
+To test this scenario we have available one example of a protein PDB file - [download it here](inductiva/resources/alanine.pdb).
 
 ```python
  import inductiva
 
  # Initialize the scenario
  scenario = inductiva.molecules.scenarios.ProteinSolvation(
-     protein_pdb = "protein_pdb_file.pdb"
+     protein_pdb = "protein_pdb_file.pdb",
```
Comment (hpenedones) [inline README.md:53]: maybe rename to alanine.pdb as well?

File: inductiva/molecules/scenarios/protein_solvation/README.md [hunk 2]
```diff
@@ -31,16 +31,18 @@ Moreover, the hardware and interaction are configured with the usual general par
...
-     protein_pdb = "alanine.pdb"
+     protein_pdb = "alanine.pdb",
      temperature = 300)
 
  # Run a simulation
- task = scenario.simulate(simulation_time_ns = 20)
+ task = scenario.simulate(simulation_time_ns = 0.5)
 
  # Get the simulation output on your local machine.
  output = task.get_output()
```
Comment (hpenedones) [inline inductiva/molecules/scenarios/protein_solvation/README.md:48]: The images below will be outdated / not matching alanine... let's try to use alanine images, or alternatively provide as an example the protein that we rendered there...

File: README.md [hunk 3]
```diff
@@ -43,18 +43,18 @@ And you are good to go! You can start [exploring our tutorial notebooks](https:/
...
      temperature = 300)
 
  # Run a simulation
- task = scenario.simulate(simulation_time_ns = 20)
+ task = scenario.simulate(simulation_time_ns = 0.5)
```
Comment (IvanPombo) [inline README.md:57]: Is this a small simulation? Having troubles finding a small simulation_time to not have to wait to long. When copy-pasting this will go to the default machines with 4 cores, so we should have those times in mind for people to test and experiment quickly.

File: README.md [hunk 4]
```diff
@@ -43,18 +43,18 @@ And you are good to go! You can start [exploring our tutorial notebooks](https:/
 
 ### Example
 
-Example of how to use a scenario:
+To test this scenario we have available one example of a protein PDB file - [download it here](inductiva/resources/alanine.pdb).
 
 ```python
  import inductiva
```
Comment (IvanPombo) [inline README.md:50]: Add here the API key indication.

File: inductiva/molecules/scenarios/protein_solvation/README.md [hunk 5]
```diff
@@ -31,16 +31,18 @@ Moreover, the hardware and interaction are configured with the usual general par
 
 ### Example
 
+To test this scenario we have available one example of a protein PDB file - [download it here](inductiva/resources/alanine.pdb).
+
 ```python
  import inductiva
```
Comment (IvanPombo) [inline inductiva/molecules/scenarios/protein_solvation/README.md:38]: Same as above.

**Review by AugustoPeres — APPROVED** [review]
Nothing to add the other comments

File: resources/media/alanine_traject.gif
Comment (hpenedones) [inline resources/media/alanine_traject.gif:1]: seems a little too low resolution, and it has grey dots on the background... is there a better image?

File: inductiva/templates/protein_solvation/gromacs/commands.json
Comment (hpenedones) [inline inductiva/templates/protein_solvation/gromacs/commands.json:1]: The PR mentions changes to the code snippets, but this also changes the GROMACS commands being run... Can you comment on that (ideally it would go in a separate PR...)?

Comment (IvanPombo) [reply inductiva/templates/protein_solvation/gromacs/commands.json:1]: Moreover, never do a change not accounted for in the previous reviews, when someone has approved, and then merge. Technically, they have approved your previous changes, not a sneaky one.

Comment (david451lima) [reply inductiva/templates/protein_solvation/gromacs/commands.json:1]: The commands were not supposed to be present in this PR, my bad.
I will revert the changes
